python/src/vmaf/core/train_test_model.py

```diff
@@ -1108,12 +1108,16 @@ def _get_num_models_from_param_dict(cls, param_dict):
             'num_models'] if 'num_models' in param_dict else cls.DEFAULT_NUM_MODELS
         return num_models
 
+    def get_num_type_bootstrap_models(self):
```

Comment:
Don't think it is a good idea to pass out a string.

Reply:
Addressed this in the updated PR, we just count the number of models from the size of prediction_all_models.